lib/eventsource.dart

```diff
@@ -53,20 +53,25 @@ class EventSource extends Stream<Event> {
   Duration _retryDelay = const Duration(milliseconds: 3000);
   String _lastEventId;
   EventSourceDecoder _decoder;
+  Object _body;
```

Comment:
This is still an `Object` instead of a `String`. I know that a String is a subclass of Object, but it should really be String here :)

Reply:
Oh sorry! I didn't see it (typically workflow when you want to make things fast, three commits just one change :P)